### PR TITLE
add fangmin to the developers section

### PR DESCRIPTION
**Pull request by lvfangmin**

### CODE REVIEW

**Comment by Randgalt** [comment]
LGTM - please merge and push this yourself as an exercise.

File: pom.xml [hunk 1]
```diff
@@ -277,6 +277,16 @@
             <timezone>-8</timezone>
             <url>https://people.apache.org/~enis</url>
         </developer>
+
+        <developer>
+            <name>Fangmin Lyu</name>
+            <email>fangmin@apache.org</email>
+            <roles>
+                <role>PMC Member</role>
```
Comment (Randgalt) [inline pom.xml]: You are also a  <role>Committer</role>

Comment (lvfangmin) [inline pom.xml]: @Randgalt to be clear, you mean I can commit without have this being reviewed?

**Comment by Randgalt** [comment]
@lvfangmin everything OK here? Add yourself as a committer too and then we can merge this and you can build the website.

File: pom.xml (hunk 1 above)
Comment (Randgalt) [inline pom.xml]: No, I meant you need to add another line to the file:

```xml
<role>PMC Member</role>
```

**Comment by lvfangmin** [comment]
@Randgalt added the Committer role :)

**Comment by Randgalt** [comment]
great :D Now you can merge this and push it so you can learn how. Then, please build the website.

**Comment by Randgalt** [comment]
BTW - once this is merged into master, do a merge of master into CURATOR-3.0

```
git checkout CURATOR-3.0
git merge master
```

Right now we keep to active branches, master and CURATOR-3.0

**Comment by lvfangmin** [comment]
@Randgalt thanks for the comment, I was just wondering how we manage the CURATOR-3.0, always keep the same as master?

**Comment by Randgalt** [comment]
I think we made a mistake a while back - master corresponds to Curator 2.x. CURATOR-3.0 corresponds to Curator 3.x. This was meant to track ZooKeeper 3.5 but we didn't realize how many years it would be in alpha. ZooKeeper 3.5.x is now probably the dominant version. Anyway, this needs to be sorted out in the future.

**Comment by lvfangmin** [comment]
yeah, ZooKeeper is moving really slow, thanks for the background, I've merged the master into CURATOR-3.0, going to build the website and publish it. (Was building based on master, and found lots of changes, since we were building based on CURATOR-3.0).

**Comment by lvfangmin** [comment]
@Randgalt I got the following error when compile the code on CURATOR-3.0 branch, is it a known issue, or it's because of my build environment issue:

[ERROR] Failed to execute goal org.codehaus.mojo:clirr-maven-plugin:2.6.1:check (default) on project curator-x-async: Execution default of goal org.codehaus.mojo:clirr-maven-plugin:2.6.1:check failed: Invalid byte tag in constant pool: 15

**Comment by lvfangmin** [comment]
Figured out, Clirr maven plugin can fail on Java 8, the bug is fixed in version 2.8, I'll provide a PR.

**Pull request merged**